### PR TITLE
Jue git assignment

### PR DESCRIPTION
**Pull request by KristinJJ**
Modified existing README.md file, created new folder/directory named quotes, and added three new files (Hamlet.md, AMidsummerNightsDream.md, and AsYouLikeIt.md).

### CODE REVIEW

File: quotes/Hamlet.md [hunk 1]
```diff
@@ -0,0 +1 @@
+“To be, or not to be; that is the question; Whether ‘tis nobler in the mind to suffer; The slings and arrows of outrageous fortune, Or to take arms against a sea of troubles” - Hamlet
```
Comment (KristinJJ) [inline quotes/Hamlet.md:1]: Classic Hamlet...

File: README.md [hunk 2]
```diff
@@ -5,7 +5,9 @@ This project is intended to help cement basic concepts around Git. It was orgina
...
+"The infinite monkey theorem states that a monkey hitting keys at random on a typewriter keyboard for an infinite amount of time will almost surely type any given text, such as the complete works of William Shakespeare. In fact, the monkey would almost surely type every possible finite text an infinite number of times. However, the probability that monkeys filling the observable universe would type a complete work such as Shakespeare's Hamlet is so tiny that the chance of it occurring during a period of time hundreds of thousands of orders of magnitude longer than the age of the universe is extremely low (but technically not zero). " 
+
+“In this context, "almost surely" is a mathematical term with a precise meaning, and the "monkey" is not an actual monkey, but a metaphor for an abstract device that produces an endless random sequence of letters and symbols. One of the earliest instances of the use of the "monkey metaphor" is that of French mathematician Émile Borel in 1913, but the first instance may have been even earlier.” - Wikipedia
```
Comment (CourtneyHoppus) [inline README.md:10]: Trying out in-line commenting.

**Review by CourtneyHoppus — COMMENTED** [review]
Test review on code.

**Comment by Annelisebx** [comment]
Hi Kristin Jue! I used to have a friend named Kristin Drew... anyway, this reminds me of my 9th grade English class when we had to memorize an entire Shakespeare scene and perform it in front of the whole class.

File: quotes/Hamlet.md (hunk 1 above)
Comment (Annelisebx) [reply quotes/Hamlet.md:1]: Commenting again...

**Pull request closed without merging**